ASP.NET_CA_Project/wwwroot/js/site.js

```diff
@@ -82,19 +82,22 @@ $(document).ready(function () {
     $(".btn-remove-cart").click(function (e) {
         e.preventDefault();
 
-        let itemId = $(this).attr("data-itemId");
+        var userResponse = window.confirm("Removing item, are you sure?");
+        if (userResponse) {
 
-        $.ajax({
-            type: "POST",
-            url: "/Cart/RemoveItem",
-            data: { itemId: itemId },
-            success: function (response) {
-                console.log("Item removed successfully!");
-            },
-            error: function (xhr, status, error) {
-                console.error("Error removing item from cart:", error);
-            }
-        })
-        location.reload();
+            let itemId = $(this).attr("data-itemId");
+            $.ajax({
+                type: "POST",
+                url: "/Cart/RemoveItem",
+                data: { itemId: itemId },
+                success: function (response) {
+                    console.log("Item removed successfully!");
+                },
+                error: function (xhr, status, error) {
+                    console.error("Error removing item from cart:", error);
+                }
+            })
+            location.reload();
```

Comment:
Need to move this under the   `success: ` function, otherwise you will have the bug where you need to click twice to see the page change